package.json

```diff
@@ -37,10 +37,11 @@
     "@babel/generator": "^7.4.0",
     "@babel/plugin-transform-block-scoping": "^7.4.0",
     "@babel/polyfill": "^7.4.0",
+    "@babel/preset-env": "^7.4.0",
     "@babel/traverse": "^7.4.0",
     "@babel/types": "^7.4.0",
-    "@babel/preset-env": "^7.4.0",
     "@octokit/rest": "^15.18.0",
+    "@qooxdoo/framework": "file:../qooxdoo",
```

Comment:
Is that what you want? Looks like you're using a git checkout here....

Reply:
You are right - it's the test code. Must be changed before merge. First we need the qooxxdo change.